src/themes/uswitch-rebrand/theme.json

```diff
@@ -1119,221 +1325,410 @@
         }
       }
     },
-
-    "collectionTabs": {
-      "base": {
-        "container": {
-          "h6": {
-            "py": "xs",
-            "fontSize": ["base", 14, "sm"],
-            "color": "grey-60"
-          }
-        }
+    "cookie-banner": {
+      "heading": {
+        "mt": [
+          0,
+          "md",
+          "md"
+        ]
       },
-      "variants": {
-        "isActive": {
-          "container": {
-            "h6": {
-              "py": "xs",
-              "borderBottomStyle": "solid",
-              "borderBottomWidth": 2,
-              "borderBottomColor": "black",
-              "fontSize": ["base", 14, "sm"],
-              "color": "black"
-            }
-          }
-        },
-        "tabSpacing": {
-          "mr": ["md", "lg"],
-          "mb": 0
-        },
-        "tabContentSpacing": {
-          "my": "xxl"
-        },
-        "overflowBorderLeft": {
-          "position": "absolute",
-          "display": "block",
-          "width": "20px",
-          "top": 0,
-          "left": 0,
-          "height": "100%",
-          "background": "linear-gradient(to right, rgba(255, 255, 255, 255) 0%, rgba(0, 0, 0, 0) 100%)",
-          "zIndex": 10
-        },
-        "overflowBorderRight": {
-          "position": "absolute",
-          "display": "block",
-          "width": "20px",
-          "top": 0,
-          "right": 0,
-          "height": "100%",
-          "background": "linear-gradient(to left, rgba(255, 255, 255, 255) 0%, rgba(0, 0, 0, 0) 100%)"
-        },
-        "borderBottom": {
-          "borderBottomWidth": 1,
-          "borderBottomStyle": "solid",
-          "borderBottomColor": "grey-20"
-        }
+      "hr": {
+        "border": 0,
+        "height": 1,
+        "bg": "grey-20"
```

Comment:
Ditto above, I think there are a few other places where `bg/background` needs updating to `backgroundColor` too.